setup.py

```diff
@@ -2,10 +2,17 @@
 
 from setuptools import setup, find_packages
 
+
+def readfile(filename):
+    with open(filename, 'r') as open_file:
+        return open_file.read()
+
+
 setup(
     name='django-fsm-log',
     version='1.7.0dev',
     description='Logging for django-fsm',
+    long_description=readfile('README.md'),
```

Comment:
The new package 2.1 spec allows you to indicate the content type of the long description. I would also add the following to indicate how PyPi should parse the long format:

```python
long_description_content_type='text/markdown',
```

Reply:
Thanks for the tip! I didn't know it.